doc/source/drivers/raster/heif.rst

```diff
@@ -31,6 +34,26 @@ Driver capabilities
 
 .. supports_virtualio:: if libheif >= 1.4
 
+Creation support
```

Comment:
to fix "AssertionError: Driver HEIF declares DCAP_CREATECOPY but doc does not!" in https://github.com/OSGeo/gdal/actions/runs/9139007180/job/25130835649?pr=8907:

```suggestion

.. supports_createcopy::

Creation support
```

Reply:
Thanks. Already fixed in a separate work branch. Still not sure what is going on with the bus error on the Alpine builds....

Reply:
> on the Alpine builds....

perhaps a issue specific with the libheif version they ship

A way to reproduce locally is to use the Docker alpine:edge image

docker run --name gdal_alpine -it -v /path/to/your/GDAL/checkout:/gdal alpine:edge

execute the apk add line at https://github.com/OSGeo/gdal/blob/master/.github/workflows/alpine/Dockerfile.ci#L3
cd /gdal
python3 -m pip install --break-system-packages -U -r autotest/requirements.txt
mkdir build_alpine
cd build_alpine
do the build as in https://github.com/OSGeo/gdal/blob/master/.github/workflows/alpine/build.sh#L16 (drop the line "-DADD_EXTERNAL_DEFERRED_PLUGIN_FOO=/tmp/foo.cpp" to simplify things)